source/examples/xgboost-azure-mnmg-daskcloudprovider/notebook.ipynb

```diff
@@ -0,0 +1,2912 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        future = client.submit(predict_model, "/tmp/model.dat", X_infer, workers=[worker])
</pre><p>@jacobthomilson is this the right approach to gather prediction results from all workers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Reply:
<p>This error suggests that <code>xgb.Booster(model_file=model_data)</code> can't load the model file. Before you try and distribute it to the workers I would just try and run this in the notebook and make sure it can read the model.</p><p> </p><p>Then we should think about what we are trying to achieve here. This cell loops over all the workers and specifically gets each one to run the same prediction on <code>X_infer</code>. I think actually what we want to do is to distribute <code>X_infer</code> to the workers and then map <code>predict_model</code> over the chunks. Do you know what type <code>X_infer</code> is? </p>